ansible_collections/arista/avd/molecule/evpn_underlay_rfc5549_overlay_ebgp/intended/configs/DC1-BL1A.cfg

```diff
@@ -250,7 +250,6 @@ router bgp 65104
       neighbor 123.1.1.10 update-source Loopback123
       neighbor 123.1.1.10 route-map RM-Tenant_A_WAN_Zone-123.1.1.10-SET-NEXT-HOP-OUT out
       neighbor fd5a:fe45:8831:06c5::a remote-as 12345
-      neighbor fd5a:fe45:8831:06c5::a send-community None
```

Comment:
Change is a fix of `null` being printed as `None` into the structured config, and therefore being taken as a string `"None"` in eos_cli_config_gen.